examples/docs/public/index.css

```diff
@@ -1,5 +1,5 @@
-@import './theme';
-@import './code';
+@import './../theme.css';
```

Comment:
`./../theme.css` is the same as `../theme.css` btw, you might rewrite it as that.. this looks like a bug imo in general though..

Reply:
@palmiak can you update this?